src/rapidhtml/tags.py

```diff
@@ -179,6 +222,15 @@ def add_head(self, *head: "BaseTag"):
                 new_head = head
             self.tags.insert(0, Head(*new_head))
 
+    def add_tag(self, *tag: "BaseTag") -> None:
+        """ """
+        if self.validate_cyclical_references(*tag):
+            self.tags.extend(tag)
+
+    def add_attr(self, **attrs) -> None:
+        """ """
+        self.attrs.update(attrs)
```

Comment:
Wonder if it's worth doing some sort of check to make sure the attribute is valid. Probably more work than it's worth

Reply:
This is a very expensive check to be honest, there are lots of core html attributes and if you plan to use some JS based framework then it blows up even larger. I shall skip the check for now